_topic_map.yml

```diff
@@ -804,7 +804,7 @@ Topics:
     File: configuring-registry-storage-gcp-user-infrastructure
 #  - Name: Configuring the registry for OpenStack user-provisioned infrastructure
 #    File: configuring-registry-storage-openstack-user-infrastructure
-  - Name: Configuring the registry for Azure user-provisioned infrastructure
+  - Name: Configuring the registry for Azure
```

Comment:
I noticed the gov section also has a prereq for a cluster with UPI. Should the `user-provisioned infrastructure` be added back here?